plugins/modules/k8s_log.py

```diff
@@ -54,6 +54,11 @@
     - If there is more than one container, this option is required.
     required: no
     type: str
+  since_seconds:
+    description:
+    - A relative time in seconds before the current time from which to show logs.
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
done!